tests/api/User/PermissionTest.php

```diff
@@ -45,7 +45,7 @@ public function listPermission()
         $I->seeResponseByJsonPathContainsJson(sprintf('$[?(@.label=="%s")]', DirectoryEnumType::DEFAULT), ['_links' => [
             'sort_list' => [],
             'create_item' => [],
-            'edit_list' => [],
+            //'edit_list' => [],
```

Comment:
May be delete this string with comment.

Reply:
I will revert this change after release.